saviSimulation/savi/asl/working.asl

```diff
@@ -58,14 +58,67 @@ targetFar :-
 	target(_,_,_,_,RANGE,_) &
 	proximityThreshold(CLOSE) &
 	(RANGE > CLOSE).
+
```

Comment:
we are likely at the point where this should be called ugv.asl instead of working.asl